e2e/scenario/scenario_ubuntu2204-standalone-containerd.go

```diff
@@ -0,0 +1,72 @@
+package scenario
+
+import (
+	"encoding/json"
+	"os"
+	"strings"
+
+	"github.com/Azure/agentbaker/pkg/agent/datamodel"
+)
+
+func (t *Template) ubuntu2204ContainerdURL() *Scenario {
+	return &Scenario{
+		Name:        "ubuntu2204ContainerdURL",
+		Description: "Tests that a node using the Ubuntu 2204 VHD with a containerd URL can be properly bootstrapped",
+		Config: Config{
+			ClusterSelector: NetworkPluginKubenetSelector,
+			ClusterMutator:  NetworkPluginKubenetMutator,
+			VHDSelector:     t.Ubuntu2204Gen2Containerd,
+			BootstrapConfigMutator: func(nbc *datamodel.NodeBootstrappingConfiguration) {
+				nbc.ContainerService.Properties.AgentPoolProfiles[0].Distro = "aks-ubuntu-containerd-22.04-gen2"
+				nbc.AgentPoolProfile.Distro = "aks-ubuntu-containerd-22.04-gen2"
+				nbc.ContainerdPackageURL = "https://packages.microsoft.com/ubuntu/22.04/prod/pool/main/m/moby-containerd/moby-containerd_1.6.9+azure-ubuntu22.04u1_amd64.deb"
+			},
+			LiveVMValidators: []*LiveVMValidator{
+				containerdVersionValidator("1.6.9"),
+			},
+		},
+	}
+}
+
+func (t *Template) ubuntu2204ContainerdVersion() *Scenario {
```

Comment:
we usually try to separate individual scenarios into their own files for clarity